deploy/009-deploy-vtokens.ts

```diff
@@ -5,11 +5,10 @@ import { DeployResult } from "hardhat-deploy/dist/types";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
-import { getConfig, getTokenConfig } from "../helpers/deploymentConfig";
+import { blocksPerYear, getConfig, getTokenConfig } from "../helpers/deploymentConfig";
 import { InterestRateModels } from "../helpers/deploymentConfig";
 import { getUnregisteredVTokens, toAddress } from "../helpers/deploymentUtils";
-
-const ADDRESS_ONE = "0x0000000000000000000000000000000000000001";
+import { AddressOne } from "../helpers/utils";
```

Comment:
What is the purpose for this address? Is it used for testing?

Reply:
It's used to set a no zero value (skipping the usual check) to the Shortfall contract defined in the VTokens. It's a workaround. We'll deploy the Shortfall in the future, and then we'll set it in the VToken contracts, but in the meantime we can live with this value

Used [here](https://github.com/VenusProtocol/isolated-pools/pull/294/files#diff-29c765173606378963b47daa17adc16f55b2468795745e6b777f295e76fc3fb0R124)